sklearn/linear_model/perceptron.py

```diff
@@ -114,7 +114,20 @@ class Perceptron(BaseSGDClassifier):
     n_iter_ : int
         The actual number of iterations to reach the stopping criterion.
         For multiclass fits, it is the maximum over every binary fit.
-
+      
+    Example
+    --------
+    >>> from sklearn.datasets import load_digits
+    >>> from sklearn.linear_model import perceptron
+    >>> from sklearn.model_selection import train_test_split
+    >>> from sklearn.metrics import accuracy_score
+    >>> digits=load_digits()
+    >>> data=digits.data
+    >>> label=digits.target
+    >>> X_train,X_test,y_train,y_test=train_test_split(data,label,random_state=123)
+    >>> model=perceptron.Perceptron(penalty='l2',alpha=0.001,random_state=124,max_iter=600,tol=0.23)
+    >>> model.fit(X_train,y_train);
```

Comment:
Pep8: space after ,

Reply:
Please remove the ;

And I assume this fit should be followed by score or something